packages/components/menu/src/Menu.test.tsx

```diff
@@ -80,6 +80,7 @@ describe('Menu', function () {
   });
 
   it('do call onClose when selecting list item', async () => {
+    jest.useFakeTimers();
```

Comment:
wouldn't `beforeEach` work here?

Reply:
We only need this for tests using user-event. Additionally, if we put it in `beforeEach` it will also be used with the axe tests, and there's an issue with `axe-core` where it doesn't work with jest's fake timers: https://github.com/dequelabs/axe-core/issues/3055